examples/typescript/package.json

```diff
@@ -0,0 +1,14 @@
+{
+		"name": "mssql-test",
+		"devDependencies": {
+			"@types/node": "^14"
+		},
+		"dependencies": {
+			"typescript": "^4.0.0",
+			"@pulumi/pulumi": "^3.0.0",
+			"@pulumi/azure": "5.38.0",
```

Comment:
Preferrably use `"^5"` as the version identifier. You currently lock it to a specific version. So more like:

```
"@pulumi/pulumi": "^3"
"@pulumi/azure": "^5"
"@pulumi/azuread": "^3"
"@pulumi/random": "^4"
```

For the `mssql` provider itself, also use the carot character to let it pick up newer releases when they become available:

```
"@pulumiverse/mssql": "^0.0.6"
```